src/backend/catalog/gp_matview_aux.c

```diff
@@ -462,3 +462,16 @@ MatviewIsGeneralyUpToDate(Oid mvoid)
 	return ((auxform->datastatus == MV_DATA_STATUS_UP_TO_DATE) || 
 			(auxform->datastatus == MV_DATA_STATUS_UP_REORGANIZED));
 }
+
```

Comment:
> namestrcpy(&mvname, get_rel_name(mvoid));

mvname without schema, maybe same `mvname` cause confused.

Reply:
Good catch.
The mvname should be with a schema prefix, this is for debug purpose during dev, maybe it could be removed, should be another topic.